lunes_cms/cms/models/document.py

```diff
@@ -54,6 +55,14 @@ class Document(models.Model):
     created_by = models.CharField(
         max_length=255, null=True, blank=True, verbose_name=_("created by")
     )
+    created_by_link = models.ForeignKey(
```

Comment:
```suggestion
    created_by = models.ForeignKey(
```
In the end, we just want one final field - it should be called `created_by` and it should be a foreign key.
You just have to watch out how to get there - just applying this suggestion isn't going to work, because it just changes the field type of the existing `created_by` field instead of using the additional information the `created_by_link` field provides.
Instead, you have to perform two steps: 1. remove the `created_by` field and create another migration for it, then 2. rename `created_by_link` to `created_by` and create yet another migration for it, too.